src/utils/interceptRequest.ts

```diff
@@ -4,9 +4,12 @@ import { MockingSchema } from '../composeMocks'
 import { SchemaEntryBody } from '../handlers/createHandler'
 import { FullMatch } from './matchPath'
 
-const postMessageToWorker = (event: MessageEvent, message: string) => {
+const messageWorker = (event: MessageEvent, message: string) => {
   const port = event.ports[0]
-  port && port.postMessage(message)
+
+  if (port) {
```

Comment:
Should handle a scenario when `port` cannot be found for some reason. Suggest page refresh in that case.

Reply:
Not a part of these changes, ignoring for now.